containers/java/clipper-java-container/src/main/java/clipper/container/app/logging/RPCEventHistory.java

```diff
@@ -0,0 +1,38 @@
+package clipper.container.app.logging;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class RPCEventHistory {
```

Comment:
No that's fine. I just wasn't sure what the purpose of the event history was (and the RPC protocol test was one of the last files I looked at when reviewing, so I saw this file first).